drivers/SmartThings/matter-sensor/profiles/air-quality-sensor-tvoc-level.yml

```diff
@@ -1,4 +1,4 @@
-name: air-quality-ikea
+name: air-quality-sensor-tvoc-level.yml
```

Comment:
remove the .yml at the end of the name, the names do not include the file extensions
```suggestion
name: air-quality-sensor-tvoc-level
```

Reply:
Fixed in 2f6d6be9343a806c29c8f9bfaa5122623aa0d116.